docs/css/_home.scss

```diff
@@ -1,39 +1,305 @@
+/* COLORS
+// -------------------------
+#46AEDA Algolia blue
+#FF2E6C Radical Red
+#091724 Black Pearl
+#C7C4C4 Gray text
+*/
+
+/* GENERAL
+// ------------------------- */
 html, body {
   position: relative;
-  @include size(100%,100%);
-  background-attachment: fixed; }
-
-h2 {
-  font-size: 1.8em;
-  font-weight: normal;
-  letter-spacing: 1px;
-  margin-bottom: 1em;
+  width: 100%;
+  height: 100%;
+  background-attachment: fixed;
+  font-family: "Raleway", "Open Sans", Helvetica, Arial, sans-serif;
 }
 
-.showcase--image {
-  max-width: 200px;
+
+
+/* HEADER
+// ------------------------- */
+header.site-header {
```

Comment:
FYI, this is an over specific selector (tagName + className). className is enough here.

Just sayin', still accepting the PR.